Command/ExpiredVideoNotificationCommand.php

```diff
@@ -42,13 +42,18 @@ private function initParameters()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->initParameters();
         $days = abs(intval($input->getArgument('days')));
         $range = $input->getArgument('range');
         if (!is_int($days)) {
             $output->writeln('Please, write an integer number');
         }
 
+        if (0 == $days) {
```

Comment:
`$days` in this command is not `pumukit_expired_video.expiration_date_days`. It is a InputArgument